src/renderers/shared/fiber/__tests__/ReactIncrementalSideEffects-test.js

```diff
@@ -1129,6 +1133,14 @@ describe('ReactIncrementalSideEffects', () => {
       null,
     ]);
 
+    expectDev(normalizeCodeLocInfo(console.error.calls.argsFor(0)[0])).toBe(
+      'Warning: Stateless function components cannot be given refs. ' +
+      'Attempts to access this ref will fail. Check the render method ' +
+      'of `FunctionalComponent`.\n' +
```

Comment:
I’m going to push one more commit with that change for review since it looks like you landed that `getCurrentFiberOwnerName` change yesterday.